cmake/precompiled_header.cmake

```diff
@@ -1,3 +1,6 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
>All rights reserved. [](start = 39, length = 20)

Why keep "All rights reserved" here and remove it for the source files?

Reply:
To make it match everywhere else.

---
In reply to: [374411856](https://github.com/microsoft/onnxruntime/pull/2970#discussion_r374411856) [](ancestors = 374411856)

Reply:
Same thing for the other .cmake files and also in CMakeLists.txt?

---
In reply to: [374412145](https://github.com/microsoft/onnxruntime/pull/2970#discussion_r374412145) [](ancestors = 374412145,374411856)

Reply:
hmm... seems like All rights reserved is more common in the code base... switching to that.

---
In reply to: [374412543](https://github.com/microsoft/onnxruntime/pull/2970#discussion_r374412543) [](ancestors = 374412543,374412145,374411856)